src/pages/Summary.jsx

```diff
@@ -165,6 +168,63 @@ const Summary = () => {
         };
     }, [user]);
 
+    // Upload unsynced data from local database
+    useEffect(() => {
+        if (!user) return;
+        const uploadUnsyncedData = async () => {
+            const unsyncedLogs = await ldb.logs.where("unsynced").equals(1).toArray();
+            if (unsyncedLogs.length === 0) return;
+
+            const keys = checkKeys();
+            if (keys === "discreet") return;
+
+            for (let log of unsyncedLogs) {
+                let data = new FormData();
+                data.append("timezone", DateTime.local().zoneName);
+                data.append("mood", log.mood);
+                data.append("journal", log.journal);
+                data.append("average", log.average);
+                data.append("keys", JSON.stringify(checkKeys()));
+                data.append("addFlag", log.addFlag ? log.addFlag + " offlineSync:" + log.timestamp : " offlineSync:" + log.timestamp);
```

Comment:
ohhh now I see the format. didn't even think of this. yeah def need a comment in the backend and here explaining this